File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_ViewModelMyCheckboxesRow.cs
/// <filename>
///     ListViews_ViewModelMyCheckboxesRow.cs
/// </filename>


namespace widgetassertions
{
    public class ListViews_ViewModelMyCheckboxesRow
  {
        private bool isHeaderCheckBoxChecked;
     
        public     virtual bool getIsHeaderCheckBoxChecked()
    {
      return this.isHeaderCheckBoxChecked;
    }
     
        private string HeaderCheckBoxText;
     
        public     virtual string getHeaderCheckBoxText()
    {
      return this.HeaderCheckBoxText;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_ViewModelMyCheckboxesRow.cpp

#include "ListViews_ViewModelMyCheckboxesRow.h"
#include "ListViews_ViewModelMyCheckboxesRow.h"
#include <string>

namespace widgetassertions
{
  bool ListViews_ViewModelMyCheckboxesRow::getIsHeaderCheckBoxChecked( ) 
  {
    return this->isHeaderCheckBoxChecked;
  }
  std::string ListViews_ViewModelMyCheckboxesRow::getHeaderCheckBoxText( ) 
  {
    return this->HeaderCheckBoxText;
  }
  int ListViews_ViewModelMyCheckboxesRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

